src/typescript-transpile.js

```diff
@@ -0,0 +1,152 @@
+// From https://github.com/teppeis/typescript-simple/blob/master/index.js
```

Comment:
Removed the js file, there is already a repo that has it (previously was waiting on a PR), so referencing that. Also ran `grunt lint`